CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/experimental/TestParametersPlugin.java

```diff
@@ -277,7 +277,7 @@ protected void updateItem(final ParameterValue item, final boolean empty) {
 
         final PluginParameter<FloatParameterValue> probability = FloatParameterType.build(PROBABILITY_PARAMETER_ID);
         probability.setName("Probability");
-        probability.setDescription("0 ≤ p ≤ 1");
+        probability.setDescription("0 = p = 1");
```

Comment:
Revert this change

Reply:
Not sure how this changed since I didn't touch this class, but I'll revert it. Do you know why it would've changed? @antares1470

Reply:
You're guess is as good as mine. I suspect it is something to do with the symbols that got swapped out (not your traditional keyboard characters)